Porting/release_managers_guide.pod

```diff
@@ -478,8 +478,17 @@ time can save you from having to work out the details during the actual
 release process.
 
 Read F<Porting/how_to_write_a_perldelta.pod>, and try to make sure that
-every section it lists is, if necessary, populated and complete. Copy
-edit the whole document.
+every section it lists is, if necessary, populated and complete.
```

Comment:
I think I would have put this section out of a this paragraph and in its own, guarded by a `=for checklist`.

But it's not a blocker to merge this change :)